### PR TITLE
Allow use of Guzzle 6.0

### PR DESCRIPTION
**Pull request by okaufmann**
I've tested it on my homestead and a windows 10 machine.

### CODE REVIEW

File: laravel [hunk 1]
```diff
@@ -7,7 +7,7 @@ if (file_exists(__DIR__.'/../../autoload.php')) {
     require __DIR__.'/vendor/autoload.php';
 }
 
-$app = new Symfony\Component\Console\Application('Laravel Installer', '1.2.1');
+$app = new Symfony\Component\Console\Application('Laravel Installer', '1.2.2');
```
Comment (GrahamCampbell) [inline laravel]: don't do that please

**Comment by GrahamCampbell** [comment]
Please squash to one commit.

File: laravel (hunk 1 above)
Comment (okaufmann) [inline laravel]: Should I leave it as it is?

Comment (vinkla) [inline laravel]: Yes, @taylorotwell or @GrahamCampbell will handle the versions.

Comment (okaufmann) [inline laravel]: Ok, sorry. Is there a Contribute Guide anywhere ?

Comment (vinkla) [inline laravel]: No worries. I don't there are any guidelines for this tool. It is common that the maintainers handle both versions and changelogs. We don't know if they want to add more stuff before the next release. We also don't know if this is going to be a major update or not.

Comment (okaufmann) [inline laravel]: U are right, thanks for make things clear ;)

**Pull request merged**